Telegram/BUILD

```diff
@@ -409,6 +433,7 @@ objc_library(
     srcs = glob([
         "Watch/Extension/**/*.m",
         "Watch/SSignalKit/**/*.m",
+        "Watch/FlickTypeKit/**/*.m",
```

Comment:
Thanks, I haven’t used Bazel before so I’m not sure what would be the best way to do this. Do you mean a separate target that only includes the FlickTypeKit source file(s)? I thought it made sense here since it’s the watch extension library, but maybe I’m misunderstanding what you mean by a separate glob filter...

Reply:
I mean it's really better for further merging to concat arrays with sources

```python
flick_type_srcs = glob(["Watch/FlickTypeKit/**/*.m"])

srcs = glob(["Watch/Extension/**/*.m",
             "Watch/SSignalKit/**/*.m",
                      # ...
             ]) +  flick_type_srcs 
```